save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/benchmarks/TabMenuBar.kt

```diff
@@ -0,0 +1,58 @@
+package com.saveourtool.save.entities.benchmarks
+
+import com.saveourtool.save.domain.Role
+
+/**
+ * Interface for tab bar in many pages
+ */
+interface TabMenuBar<T> {
+    /**
+     * Default value in every Enum classes
+     */
+    val defaultTab: T
+
+    /**
+     * Regular expression for url classification
```

Comment:
```suggestion
     * Regular expression to determine tab based on URL
```
Do I understand right? Then why not introduce a method